sdk/scheduler/src/main/java/com/mesosphere/sdk/state/SchemaVersionStore.java

```diff
@@ -35,6 +44,14 @@
 
     private final Persister persister;
 
+    public static int getSupportedSchemaVersionSingleService() {
```

Comment:
How about just `getSingleServiceVersion`? `Supported` is implied, and `Schema` is implied via this being in `SchemaVersionStore`.